staging/src/k8s.io/api/storage/v1beta1/types.go

```diff
@@ -348,6 +348,8 @@ const (
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// DEPRECATED - This group version of CSINode is deprecated by storage/v1/CSINode.
```

Comment:
Side note: I noticed the other objects in here, StorageClass, VolumeAttachment are not deprecated. Should we go ahead and deprecate them (in a separate PR)?

Reply:
Originally, the plan was to deprecate whole group-versions, but that seems to have shifted in machinery.

@liggitt or @lavalamp - thoughts?  Should we deprecate individual types from a group?  I don't think we can, really.

Reply:
I'll go ahead and deprecate the other objects in a separate pr

Reply:
I'm not sure I buy this.  At all.  Deprecation policy says:

Rule #1: API elements may only be removed by incrementing the version of the API group.